app/weddingproject/settings.py

```diff
@@ -24,9 +24,9 @@
 SECRET_KEY = config('SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = config('DEBUG', default=True, cast=bool)
```

Comment:
In my opinion, we do not need to make this change at this moment. Also the default would need to be set to False and a condition should be added in our docker-compose to change it to true.

Reply:
I pulled all from `WP-46-configure-git-hub-actions` branch to test DEBUG as an environment variable and see if it works. It worked, but since we will use `DEBUG = True` for now, then I returned to the oldest version, where `DEBUG` wasn't an environment variable.

If we want we can put `DEBUG = True` in `.env`, but set the default to `False` in `settings.py.` Then we would also have to add in `docker-compose.yml` a `DEBUG=${DEBUG}`. However, this is too troublesome. Therefore, I just changed `ALLOWED_HOSTS` in `settings.py` (see line 29) as I have explained in our last meeting. Nothing needs to be added to `.env` or to `docker-compose.yml` (besides the `SECRET_KEY`). I tested and it worked. Also the Action triggered by the push worked fine.

I think that the change to `ALLOWED_HOSTS` makes sense. We don't have to change anything in our code (not in `.env`, neither in `docker-compose.yml`). Also, we already prepare `ALLOWED_HOSTS` to be an environment variable and if we want to set some value to it, we just have to add it to `.env` and `ALLOWED_HOSTS=${ALLOWED_HOSTS}` in `docker-compose.yml`.

Please, take a look at my latest changes to this branch.